addons/mail/models/mail_tracking_value.py

```diff
@@ -117,23 +119,28 @@ def _tracking_value_format(self):
         """
         if not self:
             return []
-        field_models = self.field_id.mapped('model')
-        if len(set(field_models)) != 1:
+
+        field_models = set(self.field_id.mapped('model'))
+        if len(field_models) > 1:
             raise ValueError('All tracking value should belong to the same model.')
-        TrackedModel = self.env[field_models[0]]
-        tracked_fields = TrackedModel.fields_get(self.field_id.mapped('name'), attributes={'string', 'type'})
+        fields_sequence_map = {
+            tracking.field_info['name']: tracking.field_info.get('sequence', 100)
+            for tracking in self.filtered('field_info')
+        }
+        if field_models:
+            TrackedModel = self.env[field_models.pop()]
```

Comment:
Actually since https://github.com/odoo/odoo/pull/155034 we have to group per field model, instead of taking the first one: assumption that the model is always the same is not correct anymore (kind of multi-fix :D ) .

Reply:
But since we fail when `field_models` has more then 1 element, the model here will always be the same. The whole point of this patch is that `field_models` could be empty now. So we only handle here no model or _one_ model.

Reply:
But we should probably not fail :disguised_face:

Reply:
I pushed a (hacky) fixup that may work for multi models. (I'll remove the `and False` once it looks green, it's there only to see how it goes in runbot).